apps/server/src/shared/repo/contextexternaltool/context-external-tool.repo.ts

```diff
@@ -1,19 +1,18 @@
 import { EntityName } from '@mikro-orm/core';
 import { EntityManager } from '@mikro-orm/mongodb';
 import { Injectable } from '@nestjs/common';
-import {
-	ContextExternalTool,
-	ContextExternalToolDO,
-	ContextRef,
-	IContextExternalToolProperties,
-	SchoolExternalTool,
-	SchoolExternalToolRefDO,
-} from '@shared/domain';
-import { ContextExternalToolType } from '@shared/domain/entity/tools/course-external-tool/context-external-tool-type.enum';
 import { BaseDORepo } from '@shared/repo';
 import { LegacyLogger } from '@src/core/logger';
-import { ToolContextType } from '@src/modules/tool/common/interface/tool-context-type.enum';
 import { ContextExternalToolQuery } from '@src/modules/tool/context-external-tool/uc/dto/context-external-tool.types';
+import { SchoolExternalTool } from '@src/modules/tool/school-external-tool/entity';
```

Comment:
This repo has to be moved to tool module

Reply:
It's not part of the ticket .... 
https://ticketsystem.dbildungscloud.de/browse/N21-1053